cpp/include/cudf/text/detail/tokenize.hpp

```diff
@@ -19,35 +19,15 @@
 #include <cudf/scalar/scalar.hpp>
 #include <cudf/strings/strings_column_view.hpp>
 
-namespace cudf
-{
+using namespace cudf;
```

Comment:
This is considered bad practice. Don't make all symbols in a namespace visible (especially in a header) with `using namespace` directives.  It is OK to bring in individual names for convenience, e.g. `using cudf::table_view`.
https://google.github.io/styleguide/cppguide.html#Namespaces